sunflower-land/sunflower-land
# [FEAT] Deposit

# Description
Co-Authored by: [Romy Furtado](<25412194+Rofrtd@users.noreply.github.com>)


**NOTE: You will need to add the next deposit contract address to your `.env` to test**

This PR adds a new in-game deposit system. You can access this from the same locations you could access before. You're able to see the balances of SFL and Collectibles in your personal and add what you would like to send to your farm.

You will no longer be required to manually "refresh" to see the deposited items on your farm. 

![Screen Shot 2023-02-16 at 9 23 00 am](https://user-images.githubusercontent.com/17863697/219209114-72b60484-5384-43f9-8006-dfcb2ab53560.png)

![Screen Shot 2023-02-16 at 9 41 58 am](https://user-images.githubusercontent.com/17863697/219209170-64a48d37-a4ba-43ec-b82d-6698d4f3ccd5.png)

Fixes #issue

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Click on Balance on your farm and deposit SFL and Items.
- Go to Goblin Retreat and click on the Bank and do another deposit.

NOTE: You should not be able to access the deposit modal from the Community Garden

# Checklist:
